conf/test.config

```diff
@@ -18,4 +18,6 @@ params {
   // Input data
   input = 'https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/variants/variants.vcf'
   alleles = 'https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/alleles/alleles.txt'
-}
\ No newline at end of file
+  // Ignore `--input` as otherwise the parameter validation will throw an error
+  schema_ignore_params = 'genomes,input_paths,input'
```

Comment:
... after thinking about `schema_ignore_params` in a different context, I think this line is not needed here (and in test_full.config), since you specify `input`.
From what I understood is that `input` was added to this location in the template (in addition to `schema_ignore_params = 'genomes,input_paths'` in `nextflow.config`) because in the template only `input_paths` was specified and otherwise the parameter validation would fail since by default `input` is required. 

Probably doesn't harm here, just wanted to mention this, since it might cause confusion.

Reply:
But we're using the "alternative" input parameters, e.g. `--peptides`, in some of the tests. I guess that might be a problem, right?

Reply:
I wouldn't think so, since you removed `input` from the required parameters and thus the validation wouldn't fail.

Reply:
Makes sense 👍